apps/frontend/src/libs/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -0,0 +1,78 @@
+import { Icon } from "~/libs/components/components.js";
+import { useLocation } from "~/libs/hooks/hooks.js";
+
+import styles from "./styles.module.css";
+
+type BreadcrumbItem = {
```

Comment:
Not sure if we can have this here. Maybe it's better to create new file for this type
```suggestion

```

Reply:
Thanks for the review, i will look into moving it elsewhere so it adheres with our project structure better